demos/demo_reactive_planners_solo12_step_adjustment_walk.py

```diff
@@ -0,0 +1,317 @@
+"""Demo file for running the reactive planner on SOLO 12
+The logic is the same as the demo_reactive_planners_solo12_step_adjustment_walk.ipynb
+This file is used for ROS 2. An example of the usage of this class is here: 
+https://github.com/swuuu/ros2_controllers_solo12/blob/master/ros2_control_test_nodes/ros2_control_test_nodes/node.py
```

Comment:
The problem is that Olivier Stasse didn't want the packages containing the controllers to be in that repository: https://github.com/stack-of-tasks/ros2_control_solo/pull/1#pullrequestreview-1098610608. Perhaps we could ask him to create a new repository?

Reply:
It's totally fine. It's important to make sure it won't be multiple versions of the same code to avoid bugs:)
Create a new repository on your GitHub ros2_demo_solo for all demo nodes that we will have (or any name that you think is good ). We will add it to McGill organization GitHub in the future.